src/app/components/tokens/index.html

```diff
@@ -0,0 +1,128 @@
+<sky-demo-page
+  pageTitle="Tokens"
+  skyuxModule="SkyTokensModule"
+>
+
+  <sky-demo-page-summary>
+    <p>
+      The tokens component displays a series of specified objects that users can interact with. Users can navigate through the list of tokens and remove tokens from the list. Tokens can be also added to the list through the tokens component's message stream.
+    </p>
+  </sky-demo-page-summary>
+
+  <sky-demo-page-properties sectionHeading="Tokens list properties">
+    <sky-demo-page-property
+      propertyName="disabled"
+      defaultValue="true"
+      isOptional="true"
+    >
+      Indicates whether to disable the tokens list to prevent users from selecting tokens, dismissing tokens, or navigating through the list with the arrow keys. When the tokens list is disabled, users can still place focus on items in the list using the Tab key.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="dismissible"
+      defaultValue="true"
+    >
+      Indicates whether the tokens list is dismissable so that users can close the entire list.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="displayWith"
+      defaultValue="name"
+      isOptional="true"
+    >
+      Specifies the token property to display in the tokens list.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="focusable"
+      defaultValue="true"
+      isOptional="true"
+    >
+      Indicates whether users can focus on items in the list using the Tab key. This does not affect the ability of users to select tokens, dismiss tokens, or navigate through the list with the arrow keys.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="messageStream"
+    >
+      Specifies ...
```

Comment:
This is an observable of SkyTokenMessageTypes that can send certain commands to the tokens component. The options are multiple focusing options and one to remove the active token.
Options are: FocusLastToken, FocusActiveToken, FocusPreviousToken, FocusNextToken, and RemoveActiveToken.